### DISCUSSION TITLE
Make toolchains installed via ALR publicly visible

### DISCUSSION THREAD

**Discussion by RREE** [alire-project/alire#1058] [Ideas]
`alr toolchain` makes it easy to install various toolchains for native and cross development. I think it'd helpful if the installation could be queried for the bin dir. This way users can add it to their PATH environment. Not everybody has moved their projects to Alire yet.

I propose an option to `alr toolchain` to not show the notes but the PREFIX dir. An example output could look like this:
```
❯ alr toolchain --show-prefix
CRATE         VERSION       INSTALL_DIR (PREFIX)
gprbuild      22.0.1        /home/re/.config/alire/cache/dependencies/gprbuild_22.0.1_24dfc1b5
gprbuild      2021.0.0+0778 /usr
gnat_avr_elf  11.2.4        /home/re/.config/alire/cache/dependencies/gnat_avr_elf_11.2.4_aa9d2693
gnat_arm_elf  11.2.4        /home/re/.config/alire/cache/dependencies/gnat_arm_elf_11.2.4_ba16d45c
gnat_native   11.2.3        /home/re/.config/alire/cache/dependencies/gnat_native_11.2.3_d255cfb3
gnat_external 10.2.1        /usr
```

The inconvenience with this is that the user still has to know what to add to PATH. One has to remember to add `/bin` to the output

**Comment by Fabien-Chouteau** [comment]
@RREE You can already do that with `alr -n toolchain --install gnat_native --install-dir ~/opt/gnat/` for instance.

**Comment by RREE** [comment]
Yes, I saw that one can install into any dir including `/opt` or `/usr/local` as typical public dirs.

My point was about reusing already installed toolchains, reporting where they are installed and giving the user the possibility to add the install dir to their own PATH. 

Given today's disk sizes it is probably not worth it. One can always install a second toolchain if needed.

**Comment by Fabien-Chouteau** [comment]
> Given today's disk sizes it is probably not worth it. One can always install a second toolchain if needed.

Indeed, and I don't think the Alire project should spend much time and increase its complexity to support workflows that are not using Alire.
